src/lamp_py/tableau/jobs/glides.py

```diff
@@ -0,0 +1,249 @@
+        for batch in ds.to_batches(batch_size=500_000):
+            polars_df = pl.from_arrow(batch)
+
+            if not isinstance(polars_df, pl.DataFrame):
+                raise TypeError(f"Expected a Polars DataFrame or Series, but got {type(polars_df)}")
+            polars_df = polars_df.with_columns(
+                pl.col("data.metadata.inputTimestamp")
+                .str.strptime(pl.Datetime, "%Y-%m-%dT%H:%M:%SZ", strict=False)
+                .dt.convert_time_zone(time_zone="US/Eastern")
+                .dt.replace_time_zone(None),
+                pl.col("data.signedInAt")
+                .str.strptime(pl.Datetime, "%Y-%m-%dT%H:%M:%SZ", strict=False)
+                .dt.convert_time_zone(time_zone="US/Eastern")
+                .dt.replace_time_zone(None),
+                pl.col("time").dt.convert_time_zone(time_zone="US/Eastern").dt.replace_time_zone(None),
+            )
+            writer.write_table(polars_df.to_arrow())
+
+
+class HyperGlidesTripUpdates(HyperJob):
+    """HyperJob for Trip Updates Glides Data"""
+
+    def __init__(self) -> None:
+        HyperJob.__init__(
+            self,
+            hyper_file_name=tableau_glides_all_trips_updated.prefix.rsplit("/")[-1].replace(".parquet", ".hyper"),
+            remote_parquet_path=tableau_glides_all_trips_updated.s3_uri,
+            lamp_version=tableau_glides_all_trips_updated.version,
+        )
+
+        tu = TripUpdates()
+        assert glides_trips_updated_schema == tu.event_schema
+
+    @property
+    def parquet_schema(self) -> pyarrow.schema:
+        return glides_trips_updated_schema
+
+    def create_parquet(self, _: None) -> None:
+        self.update_parquet(None)
+
+    def update_parquet(self, _: None) -> bool:
+        # only run once per day after 11AM UTC - 6 or 7 AM EST
+        if object_exists(tableau_glides_all_trips_updated.s3_uri):
+            now_utc = datetime.now(tz=timezone.utc)
+            last_mod: datetime = file_list_from_s3_with_details(
+                bucket_name=tableau_glides_all_trips_updated.bucket,
+                file_prefix=tableau_glides_all_trips_updated.prefix,
+            )[0]["last_modified"]
+
+            if now_utc.day == last_mod.day or now_utc.hour < 11:
+                return False
+
+        create_trips_updated_glides_parquet(self, None)
+        return True
+
+
+class HyperGlidesOperatorSignIns(HyperJob):
+    """HyperJob for Operator Sign-Ins Glides Data"""
+
+    def __init__(self) -> None:
+        HyperJob.__init__(
+            self,
+            hyper_file_name=tableau_glides_all_operator_signed_in.prefix.rsplit("/")[-1].replace(".parquet", ".hyper"),
+            remote_parquet_path=tableau_glides_all_operator_signed_in.s3_uri,
+            lamp_version=tableau_glides_all_operator_signed_in.version,
+        )
+
+        osi = OperatorSignIns()
+        assert glides_operator_signed_in_schema == osi.event_schema
```

Comment:
This assertion will fail because the `OperatorSignIns.event_schema` is not flattened.